### PR TITLE
fix perf benchmark deployment issue

### PR DESCRIPTION
**Pull request by richardwxn**
Remove some unnecessary and wrong settings from operator overlay configs. This should fix the benchmark pipeline

### CODE REVIEW

File: perf/istio-install/operator_default.yaml [hunk 1]
```diff
@@ -168,20 +161,5 @@ spec:
...
-    nodeagent:
-      image: node-agent-k8s
-    pilot:
-      sidecar: true
-    prometheus:
-      enabled: false
     tracing:
-      enabled: true
-      jaeger:
-        enabled: true
```
Comment (carolynhu) [inline perf/istio-install/operator_default.yaml]: This is a fix only for master or also for 1.4?

**Comment by mandarjog** [comment]
@richardwxn what changed so that the pipeline broke?

Looks like you need to do a Manual cherry pick.

**Comment by richardwxn** [comment]
@mandarjog the prometheus installation part so that was why it failed when querying prometheus API. I think last time you mentioned something during your testing so that was the same issue. It should work now

**Pull request merged**